src/main/webapp/placeGuides/PlaceGuideOnMap.js

```diff
@@ -10,11 +10,8 @@ class PlaceGuideOnMap {
         .getInfoWindow(name, position, place, creator, description);
     this._marker = PlaceGuideOnMap.getMarker(placeType, name, position);
     this._highlighted = false;
-    this.closeInfoWindowOnMapClick();
-    this.toggleInfoWindowOnMarkerClick();
-    this.highlightOnMarkerDoubleClick();
+    this.toggleHighlightOnMarkerClick();
```

Comment:
nit: The name is a bit misleading as it indicates an action. What about calling it setupHighlightOnMarkerClick()?

Reply:
Changed it.